plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -109,22 +119,34 @@ def create_optuna_distribution_from_override(override: Override) -> Any:
 
     raise NotImplementedError(f"{override} is not supported by Optuna sweeper.")
 
-def get_function_from_path(import_string: str) -> Callable[[Trial], None]:
+
+def get_function_from_path(import_string: str) -> Any:
```

Comment:
I had to use 'Any' in the end since 'Callable' could not be used due to linting errors of it not being fully specified. Writing Callable[[DictConfig, Trial], None] doesn't work as mypy cannot defer the type from the getattr call. Furthermore 'isinstance(function, Callable[[DictConfig, Trial], None])' is not yet supported in Python. Generics can't be used in 'isinstance'

Reply:
> 'isinstance(function, Callable[[DictConfig, Trial], None])' is not yet supported...

FYI there is a builtin [`callable`](https://docs.python.org/3/library/functions.html#callable) function that can do part of the job :)

Reply:
> 'isinstance(function, Callable[[DictConfig, Trial], None])' is not yet supported...

FYI there is a builtin callable function that can do part of the job :)

Reply:
Thanks for the info! Indeed, I could reused the get_method :)